server/routes/offenceCodeDecisions/offenceCodeDecisionsSessionHelper.ts

```diff
@@ -0,0 +1,94 @@
+import { Request, Response } from 'express'
+import decisionTree from '../../offenceCodeDecisions/DecisionTree'
+import { DecisionType } from '../../offenceCodeDecisions/Decision'
+import { DecisionForm, SelectPrisonerData, SelectStaffData } from './decisionForm'
+
```

Comment:
This class is just a break out from the offenceCodeDecisions - as that was getting a bit busy - I though that this class could deal with the nitty gritty of knowing how to parse a DecisionForm from the post data and stick it on the session - then the offenceCodeDecision class could deal more with the higher level logic of when to stick things on the session and take them off etc.